TI-reports/2025/2025-Q1-Vulnerability-Disclosure-WG.md

```diff
@@ -0,0 +1,30 @@
+# 2025 Q1 Vulnerability Disclosure WG - Madison Oliver
+
+## Overview
+TI is progressing as expected. Engagement has remained relatively stable with slight decreases over the last 6 months. We've had external presentations and discussions with the [package url](https://github.com/package-url/purl-spec) community and passed a vote to adopt Advise as a new project.
+
+## Vulnerability Dislosures Working Group
+
+### Purpose
+The OpenSSF Vulnerability Disclosures Working Group seeks to help improve the overall security of the open source software ecosystem by helping mature and advocate well-managed vulnerability reporting and communication.
+
+### Current Status
+- Typically have 5-6 regular attendees for biweekly full WG meeting, limited additional engagement in the monthly APAC meeting
+- [Graduated](https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#to-become-graduated).
+- [To remain Graduated](https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#to-remain-graduated):
+   - [x] Have received TAC approval of the README.md per Incubating requirements above
+   - [x] Have met at least 4 times over a period of at least 2 months since becoming Incubating
+   - [x] Have at least 5 contributors from at least 3 different organizations attending regularly as recorded in meeting minutes.
+   - [x] Request TAC approval. TAC will vote to approve or provide constructive guidance.
+   - [x] TAC sponsor monitoring and consultation become optional.
+
+### Up Next
+- WG Project Board: https://github.com/orgs/ossf/projects/29
+- [Project Idea - CVD Guide for OSS Consumers](https://github.com/ossf/wg-vulnerability-disclosures/issues/115) > effort is stagnating and needs to be revived
```

Comment:
Not at all urgent, but it would be cool for the Vuln Disclosure WG to have a site like https://best.openssf.org/ or https://repos.openssf.org/ where recommendations were hosted and easily linkable.

Reply:
yes absolutely! I would love to make something like that - I have 3 similar issues tracking ~that need with ideas to add common vul disclosure references to our readme and well-known open source disclosure processes as examples elsewhere in our docs, in addition to tracking vul disclosure related policies. I would love to effectively turn these into a site like that! who do I talk to about that?
https://github.com/ossf/wg-vulnerability-disclosures/issues/151
https://github.com/ossf/wg-vulnerability-disclosures/issues/67
https://github.com/ossf/wg-vulnerability-disclosures/issues/73

Reply:
I don't remember where I saw this linked, but the information on how to set up those sites is on https://best.openssf.org/spp/Simplest-Possible-Process.